unittests/interface/interface_functions/test_front_end_common_graph_binary_gatherer.py

```diff
@@ -89,16 +88,21 @@ def test_mixed_binaries(self):
         """
 
         vertex_1 = _TestVertexWithBinary(
-            "test.aplx", ExecutableStartType.RUNNING)
+            "test.aplx", ExecutableType.RUNNING)
         vertex_2 = _TestVertexWithBinary(
-            "test2.aplx", ExecutableStartType.SYNC)
+            "test2.aplx", ExecutableType.SYNC)
+
+        placements = Placements(placements=[
+            Placement(vertex_1, 0, 0, 0),
+            Placement(vertex_2, 0, 0, 1)])
 
         graph = MachineGraph("Test")
         graph.add_vertices([vertex_1, vertex_2])
 
         gatherer = LocateExecutableStartType()
-        with self.assertRaises(ConfigurationException):
-            gatherer.__call__(graph)
+        results = gatherer.__call__(graph, placements=placements)
+        self.assertIn(ExecutableType.RUNNING, results)
+        self.assertIn(ExecutableType.SYNC, results)
```

Comment:
May also want to assert that these are the only things in there...

Reply:
rubebred